docker/docker-compose.tests.yml

```diff
@@ -9,6 +9,8 @@ services:
     build:
       context: ../
       dockerfile: docker/Dockerfile.test-database
+    environment:
+      MB_IMPORT_DUMPS: "false"
```

Comment:
This variable `MB_IMPORT_DUMPS` being set to `false` works as it actually runs the bash command `false`. However, as you suggested, that an undefined variable should do what is expected and the [issues](https://stackoverflow.com/questions/2953646/how-to-declare-and-use-boolean-variables-in-shell-script/21210966) with the my current syntax, I have changed the syntax to `if ["$MB_IMPORT_DUMPS" = "true"]`. :sweat_smile: